app/README.md

```
@@ -7,28 +7,39 @@ It currently supports three modes:
+npm install
+```
 
 ### Build WebApp
-```
+
+```sh
 npm run start
 ```
 
-The application will start running at http://localhost:3000. 
+The application will start running at http://localhost:3000.
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)